src/components/ArticleList.jsx

```diff
@@ -6,10 +8,7 @@ function ArticleList({ articles, isError, isLoading }) {
   }
 
   if (isLoading) {
-    return (
-      <>Loading...</>
-      // <Lottie className="loading-animation" animationData={hamsterAnimation} />
-    );
+    return <Lottie animationData={groovyWalkAnimation} loop={true} />;
```

Comment:
This looks very cool on the home page, I wasn't sure whether the page was loading at first however. It might be good to also have some text to tell a user the page is loading, because the animation itself doesn't necessarily signify a loading state